packages/vscode-extension/messages.json

```diff
@@ -14,6 +14,15 @@
 	"COMMAND_SUCCESS": "'{0}' has been successfully executed.",
 	"COMMAND_TRIGGERED": "'{0}' has been triggered.",
 
+	"CREATE_FILE_ERRORS_MISSING_VALID_FOLDER_FOR_SUITECRIPT_FILE": "No valid folder found to create a SuiteScript file. Use '{0}' from a valid SuiteCloud project structure. For more information about the project structure, see {1}",
```

Comment:
Maybe some of these sentences needs a final dot.

Reply:
The final argument is web link.
I have somehow reutilized the one from ```"MANAGE_ACCOUNTS_ERROR_MISSING_MANIFEST": "No manifest.xml file was found in {0}. Use '{1}' from an active file in a valid project folder. For more information about the project structure, see {2}",``` and it has not dot at the end neither.